1-python-wrapper/wrapper/main.py

```diff
@@ -0,0 +1,38 @@
+from flask import Flask, jsonify, render_template, request
+from model.model import predict as m_predict, load_most_recent_model, load_json
+import pandas as pd
+import json
+import logging
+
+
+logger = logging.getLogger(__name__)
+
+app = Flask(__name__)
+
+@app.route('/predict', methods=["GET", "POST"])
```

Comment:
If methods only contains `POST`, you could get rid of the `if` statement in the function body.

Reply:
Yes, for sure. Will clean up

Reply:
Oh btw, this was ment to be the case. I was planning on returning some html, dedicating the schema of the json to post.

Reply:
In that case I'd either go for standardized documentation like swagger/openapi, or omit it altogether.

Reply:
then for now I will omit.